pillow dependency in /hardware fails to install on arm64

# Overview

`make -C hardware setup` fails on arm64 machine.
computer setup per [DEV_SETUP.md](https://github.com/Opentrons/opentrons/blob/edge/DEV_SETUP.md)

```
make -C hardware setup
make[2]: Entering directory '/Users/johnmcvey/github/opentrons/opentrons/hardware'
python -m pipenv sync --dev --python 3.7
Virtualenv already exists!
Removing existing virtualenv...
Creating a virtualenv for this project...
Pipfile: /Users/johnmcvey/github/opentrons/opentrons/hardware/Pipfile
Using /Users/johnmcvey/.pyenv/versions/3.7.13/bin/python3.7m (3.7.13) to create virtualenv...
⠸ Creating virtual environment...created virtual environment CPython3.7.13.final.0-64 in 224ms
  creator CPython3Posix(dest=/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW, clear=False, no_vcs_ignore=False, global=False)
  seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/Users/johnmcvey/Library/Application Support/virtualenv)
    added seed packages: pip==22.0.4, setuptools==62.1.0, wheel==0.37.1
  activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator

✔ Successfully created virtual environment!
Virtualenv location: /Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW
Installing dependencies from Pipfile.lock (9ab443)...
An error occurred while installing pillow==9.1.0; python_version >= '3.7' --hash=sha256:eb1b89b11256b5b6cad5e7593f9061ac4624f7651f7a8eb4dfa37caa1dfaa4d0 --hash=sha256:463acf531f5d0925ca55904fa668bb3461c3ef6bc779e1d6d8a488092bdee378 --hash=sha256:5a3ecc026ea0e14d0ad7cd990ea7f48bfcb3eb4271034657dc9d06933c6629a7 --hash=sha256:c7110ec1701b0bf8df569a7592a196c9d07c764a0a74f65471ea56816f10e2c8 --hash=sha256:17869489de2fce6c36690a0c721bd3db176194af5f39249c1ac56d0bb0fcc512 --hash=sha256:c870193cce4b76713a2b29be5d8327c8ccbe0d4a49bc22968aa1e680930f5581 --hash=sha256:01ce45deec9df310cbbee11104bae1a2a43308dd9c317f99235b6d3080ddd66e --hash=sha256:5cfca31ab4c13552a0f354c87fbd7f162a4fafd25e6b521bba93a57fe6a3700a --hash=sha256:f401ed2bbb155e1ade150ccc63db1a4f6c1909d3d378f7d1235a44e90d75fb97 --hash=sha256:8782189c796eff29dbb37dd87afa4ad4d40fc90b2742704f94812851b725964b --hash=sha256:c5b0ff59785d93b3437c3703e3c64c178aabada51dea2a7f2c5eccf1bcf565a3 --hash=sha256:de344bcf6e2463bb25179d74d6e7989e375f906bcec8cb86edb8b12acbc7dfef --hash=sha256:af79d3fde1fc2e33561166d62e3b63f0cc3e47b5a3a2e5fea40d4917754734ea --hash=sha256:55dd1cf09a1fd7c7b78425967aacae9b0d70125f7d3ab973fadc7b5abc3de652 --hash=sha256:3f42364485bfdab19c1373b5cd62f7c5ab7cc052e19644862ec8f15bb8af289e --hash=sha256:69e5ddc609230d4408277af135c5b5c8fe7a54b2bdb8ad7c5100b86b3aab04c6 --hash=sha256:3fddcdb619ba04491e8f771636583a7cc5a5051cd193ff1aa1ee8616d2a692c5 --hash=sha256:7a053bd4d65a3294b153bdd7724dce864a1d548416a5ef61f6d03bf149205160 --hash=sha256:255c9d69754a4c90b0ee484967fc8818c7ff8311c6dddcc43a4340e10cd1636a --hash=sha256:6b6d4050b208c8ff886fd3db6690bf04f9a48749d78b41b7a5bf24c236ab0165 --hash=sha256:21dee8466b42912335151d24c1665fcf44dc2ee47e021d233a40c3ca5adae59c --hash=sha256:25023a6209a4d7c42154073144608c9a71d3512b648a2f5d4465182cb93d3477 --hash=sha256:c9efef876c21788366ea1f50ecb39d5d6f65febe25ad1d4c0b8dff98843ac244 --hash=sha256:a336a4f74baf67e26f3acc4d61c913e378e931817cd1e2ef4dfb79d3e051b481 --hash=sha256:66822d01e82506a19407d1afc104c3fcea3b81d5eb11485e593ad6b8492f995a --hash=sha256:4fe29a070de394e449fd88ebe1624d1e2d7ddeed4c12e0b31624561b58948d9a --hash=sha256:a598d8830f6ef5501002ae85c7dbfcd9c27cc4efc02a1989369303ba85573e58 --hash=sha256:97bda660702a856c2c9e12ec26fc6d187631ddfd896ff685814ab21ef0597033 --hash=sha256:0c51cb9edac8a5abd069fd0758ac0a8bfe52c261ee0e330f363548aca6893595 --hash=sha256:a325ac71914c5c043fa50441b36606e64a10cd262de12f7a179620f579752ff8 --hash=sha256:a5eaf3b42df2bcda61c53a742ee2c6e63f777d0e085bbc6b2ab7ed57deb13db7 --hash=sha256:35be4a9f65441d9982240e6966c1eaa1c654c4e5e931eaf580130409e31804d4 --hash=sha256:82283af99c1c3a5ba1da44c67296d5aad19f11c535b551a5ae55328a317ce331 --hash=sha256:aea7ce61328e15943d7b9eaca87e81f7c62ff90f669116f857262e9da4057ba3 --hash=sha256:fb89397013cf302f282f0fc998bb7abf11d49dcff72c8ecb320f76ea6e2c5717 --hash=sha256:c24f718f9dd73bb2b31a6201e6db5ea4a61fdd1d1c200f43ee585fc6dcd21b34 --hash=sha256:ed742214068efa95e9844c2d9129e209ed63f61baa4d54dbf4cf8b5e2d30ccf2 --hash=sha256:8d79c6f468215d1a8415aa53d9868a6b40c4682165b8cb62a221b1baa47db458! Will try again.
  🐍   ▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉ 49/49 — 00:03:02
Installing initially failed dependencies...
[InstallError]:   File "/Users/johnmcvey/.pyenv/versions/3.7.13/lib/python3.7/site-packages/pipenv/cli/command.py", line 696, in sync
[InstallError]:       system=state.system
[InstallError]:   File "/Users/johnmcvey/.pyenv/versions/3.7.13/lib/python3.7/site-packages/pipenv/core.py", line 2892, in do_sync
[InstallError]:       system=system,
[InstallError]:   File "/Users/johnmcvey/.pyenv/versions/3.7.13/lib/python3.7/site-packages/pipenv/core.py", line 1312, in do_init
[InstallError]:       pypi_mirror=pypi_mirror,
[InstallError]:   File "/Users/johnmcvey/.pyenv/versions/3.7.13/lib/python3.7/site-packages/pipenv/core.py", line 900, in do_install_dependencies
[InstallError]:       retry_list, procs, failed_deps_queue, requirements_dir, **install_kwargs
[InstallError]:   File "/Users/johnmcvey/.pyenv/versions/3.7.13/lib/python3.7/site-packages/pipenv/core.py", line 796, in batch_install
[InstallError]:       _cleanup_procs(procs, failed_deps_queue, retry=retry)
[InstallError]:   File "/Users/johnmcvey/.pyenv/versions/3.7.13/lib/python3.7/site-packages/pipenv/core.py", line 703, in _cleanup_procs
[InstallError]:       raise exceptions.InstallError(c.dep.name, extra=err_lines)
[pipenv.exceptions.InstallError]: Collecting pillow==9.1.0
[pipenv.exceptions.InstallError]:   Using cached Pillow-9.1.0.tar.gz (49.8 MB)
[pipenv.exceptions.InstallError]:   Preparing metadata (setup.py): started
[pipenv.exceptions.InstallError]:   Preparing metadata (setup.py): finished with status 'done'
[pipenv.exceptions.InstallError]: Building wheels for collected packages: pillow
[pipenv.exceptions.InstallError]:   Building wheel for pillow (setup.py): started
[pipenv.exceptions.InstallError]:   Building wheel for pillow (setup.py): finished with status 'error'
[pipenv.exceptions.InstallError]:   error: subprocess-exited-with-error
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:   × python setup.py bdist_wheel did not run successfully.
[pipenv.exceptions.InstallError]:   │ exit code: 1
[pipenv.exceptions.InstallError]:   ╰─> [188 lines of output]
[pipenv.exceptions.InstallError]:       running bdist_wheel
[pipenv.exceptions.InstallError]:       running build
[pipenv.exceptions.InstallError]:       running build_py
[pipenv.exceptions.InstallError]:       creating build
[pipenv.exceptions.InstallError]:       creating build/lib.macosx-12.2-arm64-cpython-37
[pipenv.exceptions.InstallError]:       creating build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/MpoImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageMode.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PngImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/XbmImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PcxImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/SunImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageFile.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/SpiderImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/TarIO.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/FitsStubImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/MpegImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/BdfFontFile.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/GribStubImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageStat.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PixarImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/GimpPaletteFile.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageColor.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ContainerIO.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/MspImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/MicImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/_version.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImtImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/GifImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PalmImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageQt.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageMath.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PaletteFile.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/FontFile.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PdfParser.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ExifTags.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageCms.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/FpxImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageChops.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/BufrStubImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PSDraw.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PcdImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageFilter.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageDraw2.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImagePath.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/DcxImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/__init__.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/JpegPresets.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/Hdf5StubImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/features.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageDraw.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/GimpGradientFile.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageWin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/IcoImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/_tkinter_finder.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/EpsImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/TgaImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageMorph.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/Jpeg2KImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/WalImageFile.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PcfFontFile.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/BlpImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageTk.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/GbrImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageOps.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PdfImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageShow.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageEnhance.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/WmfImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageGrab.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/WebPImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/FliImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/TiffTags.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/CurImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/_util.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/GdImageFile.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/TiffImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/IptcImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImagePalette.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/BmpImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageTransform.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/IcnsImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/McIdasImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/FitsImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/XpmImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/DdsImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageSequence.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PyAccess.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/_binary.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/Image.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/__main__.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/XVThumbImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/SgiImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PsdImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/JpegImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageFont.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PpmImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/FtexImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       running egg_info
[pipenv.exceptions.InstallError]:       writing src/Pillow.egg-info/PKG-INFO
[pipenv.exceptions.InstallError]:       writing dependency_links to src/Pillow.egg-info/dependency_links.txt
[pipenv.exceptions.InstallError]:       writing requirements to src/Pillow.egg-info/requires.txt
[pipenv.exceptions.InstallError]:       writing top-level names to src/Pillow.egg-info/top_level.txt
[pipenv.exceptions.InstallError]:       reading manifest file 'src/Pillow.egg-info/SOURCES.txt'
[pipenv.exceptions.InstallError]:       reading manifest template 'MANIFEST.in'
[pipenv.exceptions.InstallError]:       warning: no files found matching '*.c'
[pipenv.exceptions.InstallError]:       warning: no files found matching '*.h'
[pipenv.exceptions.InstallError]:       warning: no files found matching '*.sh'
[pipenv.exceptions.InstallError]:       warning: no files found matching '*.txt'
[pipenv.exceptions.InstallError]:       warning: no previously-included files found matching '.appveyor.yml'
[pipenv.exceptions.InstallError]:       warning: no previously-included files found matching '.clang-format'
[pipenv.exceptions.InstallError]:       warning: no previously-included files found matching '.coveragerc'
[pipenv.exceptions.InstallError]:       warning: no previously-included files found matching '.editorconfig'
[pipenv.exceptions.InstallError]:       warning: no previously-included files found matching '.readthedocs.yml'
[pipenv.exceptions.InstallError]:       warning: no previously-included files found matching 'codecov.yml'
[pipenv.exceptions.InstallError]:       warning: no previously-included files matching '.git*' found anywhere in distribution
[pipenv.exceptions.InstallError]:       warning: no previously-included files matching '*.pyc' found anywhere in distribution
[pipenv.exceptions.InstallError]:       warning: no previously-included files matching '*.so' found anywhere in distribution
[pipenv.exceptions.InstallError]:       no previously-included directories found matching '.ci'
[pipenv.exceptions.InstallError]:       adding license file 'LICENSE'
[pipenv.exceptions.InstallError]:       writing manifest file 'src/Pillow.egg-info/SOURCES.txt'
[pipenv.exceptions.InstallError]:       warning: build_py: byte-compiling is disabled, skipping.
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:       running build_ext
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:       The headers or library files could not be found for jpeg,
[pipenv.exceptions.InstallError]:       a required dependency when compiling Pillow from source.
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:       Please see the install instructions at:
[pipenv.exceptions.InstallError]:          https://pillow.readthedocs.io/en/latest/installation.html
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:       Traceback (most recent call last):
[pipenv.exceptions.InstallError]:         File "/private/var/folders/r6/l11jt3f5461f4t1cc_2pmt2c0000gp/T/pip-install-rjbimtoh/pillow_288a2b4a62b14ad08257bd657605fbee/setup.py", line 996, in <module>
[pipenv.exceptions.InstallError]:           zip_safe=not (debug_build() or PLATFORM_MINGW),
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/__init__.py", line 87, in setup
[pipenv.exceptions.InstallError]:           return distutils.core.setup(**attrs)
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/_distutils/core.py", line 148, in setup
[pipenv.exceptions.InstallError]:           return run_commands(dist)
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/_distutils/core.py", line 163, in run_commands
[pipenv.exceptions.InstallError]:           dist.run_commands()
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/_distutils/dist.py", line 967, in run_commands
[pipenv.exceptions.InstallError]:           self.run_command(cmd)
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/dist.py", line 1214, in run_command
[pipenv.exceptions.InstallError]:           super().run_command(command)
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/_distutils/dist.py", line 986, in run_command
[pipenv.exceptions.InstallError]:           cmd_obj.run()
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/wheel/bdist_wheel.py", line 299, in run
[pipenv.exceptions.InstallError]:           self.run_command('build')
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/_distutils/cmd.py", line 313, in run_command
[pipenv.exceptions.InstallError]:           self.distribution.run_command(command)
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/dist.py", line 1214, in run_command
[pipenv.exceptions.InstallError]:           super().run_command(command)
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/_distutils/dist.py", line 986, in run_command
[pipenv.exceptions.InstallError]:           cmd_obj.run()
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/_distutils/command/build.py", line 136, in run
[pipenv.exceptions.InstallError]:           self.run_command(cmd_name)
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/_distutils/cmd.py", line 313, in run_command
[pipenv.exceptions.InstallError]:           self.distribution.run_command(command)
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/dist.py", line 1214, in run_command
[pipenv.exceptions.InstallError]:           super().run_command(command)
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/_distutils/dist.py", line 986, in run_command
[pipenv.exceptions.InstallError]:           cmd_obj.run()
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/command/build_ext.py", line 79, in run
[pipenv.exceptions.InstallError]:           _build_ext.run(self)
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/_distutils/command/build_ext.py", line 339, in run
[pipenv.exceptions.InstallError]:           self.build_extensions()
[pipenv.exceptions.InstallError]:         File "/private/var/folders/r6/l11jt3f5461f4t1cc_2pmt2c0000gp/T/pip-install-rjbimtoh/pillow_288a2b4a62b14ad08257bd657605fbee/setup.py", line 804, in build_extensions
[pipenv.exceptions.InstallError]:           raise RequiredDependencyException(f)
[pipenv.exceptions.InstallError]:       __main__.RequiredDependencyException: jpeg
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:       During handling of the above exception, another exception occurred:
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:       Traceback (most recent call last):
[pipenv.exceptions.InstallError]:         File "<string>", line 36, in <module>
[pipenv.exceptions.InstallError]:         File "<pip-setuptools-caller>", line 34, in <module>
[pipenv.exceptions.InstallError]:         File "/private/var/folders/r6/l11jt3f5461f4t1cc_2pmt2c0000gp/T/pip-install-rjbimtoh/pillow_288a2b4a62b14ad08257bd657605fbee/setup.py", line 1009, in <module>
[pipenv.exceptions.InstallError]:           raise RequiredDependencyException(msg)
[pipenv.exceptions.InstallError]:       __main__.RequiredDependencyException:
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:       The headers or library files could not be found for jpeg,
[pipenv.exceptions.InstallError]:       a required dependency when compiling Pillow from source.
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:       Please see the install instructions at:
[pipenv.exceptions.InstallError]:          https://pillow.readthedocs.io/en/latest/installation.html
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:       [end of output]
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:   note: This error originates from a subprocess, and is likely not a problem with pip.
[pipenv.exceptions.InstallError]:   ERROR: Failed building wheel for pillow
[pipenv.exceptions.InstallError]:   Running setup.py clean for pillow
[pipenv.exceptions.InstallError]: Failed to build pillow
[pipenv.exceptions.InstallError]: Installing collected packages: pillow
[pipenv.exceptions.InstallError]:   Running setup.py install for pillow: started
[pipenv.exceptions.InstallError]:   Running setup.py install for pillow: finished with status 'error'
[pipenv.exceptions.InstallError]:   error: subprocess-exited-with-error
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:   × Running setup.py install for pillow did not run successfully.
[pipenv.exceptions.InstallError]:   │ exit code: 1
[pipenv.exceptions.InstallError]:   ╰─> [192 lines of output]
[pipenv.exceptions.InstallError]:       running install
[pipenv.exceptions.InstallError]:       /Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/command/install.py:37: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
[pipenv.exceptions.InstallError]:         setuptools.SetuptoolsDeprecationWarning,
[pipenv.exceptions.InstallError]:       running build
[pipenv.exceptions.InstallError]:       running build_py
[pipenv.exceptions.InstallError]:       creating build
[pipenv.exceptions.InstallError]:       creating build/lib.macosx-12.2-arm64-cpython-37
[pipenv.exceptions.InstallError]:       creating build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/MpoImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageMode.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PngImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/XbmImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PcxImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/SunImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageFile.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/SpiderImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/TarIO.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/FitsStubImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/MpegImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/BdfFontFile.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/GribStubImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageStat.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PixarImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/GimpPaletteFile.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageColor.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ContainerIO.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/MspImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/MicImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/_version.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImtImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/GifImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PalmImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageQt.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageMath.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PaletteFile.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/FontFile.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PdfParser.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ExifTags.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageCms.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/FpxImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageChops.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/BufrStubImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PSDraw.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PcdImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageFilter.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageDraw2.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImagePath.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/DcxImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/__init__.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/JpegPresets.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/Hdf5StubImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/features.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageDraw.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/GimpGradientFile.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageWin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/IcoImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/_tkinter_finder.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/EpsImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/TgaImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageMorph.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/Jpeg2KImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/WalImageFile.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PcfFontFile.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/BlpImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageTk.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/GbrImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageOps.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PdfImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageShow.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageEnhance.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/WmfImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageGrab.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/WebPImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/FliImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/TiffTags.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/CurImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/_util.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/GdImageFile.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/TiffImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/IptcImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImagePalette.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/BmpImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageTransform.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/IcnsImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/McIdasImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/FitsImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/XpmImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/DdsImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageSequence.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PyAccess.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/_binary.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/Image.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/__main__.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/XVThumbImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/SgiImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PsdImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/JpegImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/ImageFont.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/PpmImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       copying src/PIL/FtexImagePlugin.py -> build/lib.macosx-12.2-arm64-cpython-37/PIL
[pipenv.exceptions.InstallError]:       running egg_info
[pipenv.exceptions.InstallError]:       writing src/Pillow.egg-info/PKG-INFO
[pipenv.exceptions.InstallError]:       writing dependency_links to src/Pillow.egg-info/dependency_links.txt
[pipenv.exceptions.InstallError]:       writing requirements to src/Pillow.egg-info/requires.txt
[pipenv.exceptions.InstallError]:       writing top-level names to src/Pillow.egg-info/top_level.txt
[pipenv.exceptions.InstallError]:       reading manifest file 'src/Pillow.egg-info/SOURCES.txt'
[pipenv.exceptions.InstallError]:       reading manifest template 'MANIFEST.in'
[pipenv.exceptions.InstallError]:       warning: no files found matching '*.c'
[pipenv.exceptions.InstallError]:       warning: no files found matching '*.h'
[pipenv.exceptions.InstallError]:       warning: no files found matching '*.sh'
[pipenv.exceptions.InstallError]:       warning: no files found matching '*.txt'
[pipenv.exceptions.InstallError]:       warning: no previously-included files found matching '.appveyor.yml'
[pipenv.exceptions.InstallError]:       warning: no previously-included files found matching '.clang-format'
[pipenv.exceptions.InstallError]:       warning: no previously-included files found matching '.coveragerc'
[pipenv.exceptions.InstallError]:       warning: no previously-included files found matching '.editorconfig'
[pipenv.exceptions.InstallError]:       warning: no previously-included files found matching '.readthedocs.yml'
[pipenv.exceptions.InstallError]:       warning: no previously-included files found matching 'codecov.yml'
[pipenv.exceptions.InstallError]:       warning: no previously-included files matching '.git*' found anywhere in distribution
[pipenv.exceptions.InstallError]:       warning: no previously-included files matching '*.pyc' found anywhere in distribution
[pipenv.exceptions.InstallError]:       warning: no previously-included files matching '*.so' found anywhere in distribution
[pipenv.exceptions.InstallError]:       no previously-included directories found matching '.ci'
[pipenv.exceptions.InstallError]:       adding license file 'LICENSE'
[pipenv.exceptions.InstallError]:       writing manifest file 'src/Pillow.egg-info/SOURCES.txt'
[pipenv.exceptions.InstallError]:       warning: build_py: byte-compiling is disabled, skipping.
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:       running build_ext
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:       The headers or library files could not be found for jpeg,
[pipenv.exceptions.InstallError]:       a required dependency when compiling Pillow from source.
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:       Please see the install instructions at:
[pipenv.exceptions.InstallError]:          https://pillow.readthedocs.io/en/latest/installation.html
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:       Traceback (most recent call last):
[pipenv.exceptions.InstallError]:         File "/private/var/folders/r6/l11jt3f5461f4t1cc_2pmt2c0000gp/T/pip-install-rjbimtoh/pillow_288a2b4a62b14ad08257bd657605fbee/setup.py", line 996, in <module>
[pipenv.exceptions.InstallError]:           zip_safe=not (debug_build() or PLATFORM_MINGW),
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/__init__.py", line 87, in setup
[pipenv.exceptions.InstallError]:           return distutils.core.setup(**attrs)
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/_distutils/core.py", line 148, in setup
[pipenv.exceptions.InstallError]:           return run_commands(dist)
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/_distutils/core.py", line 163, in run_commands
[pipenv.exceptions.InstallError]:           dist.run_commands()
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/_distutils/dist.py", line 967, in run_commands
[pipenv.exceptions.InstallError]:           self.run_command(cmd)
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/dist.py", line 1214, in run_command
[pipenv.exceptions.InstallError]:           super().run_command(command)
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/_distutils/dist.py", line 986, in run_command
[pipenv.exceptions.InstallError]:           cmd_obj.run()
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/command/install.py", line 68, in run
[pipenv.exceptions.InstallError]:           return orig.install.run(self)
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/_distutils/command/install.py", line 670, in run
[pipenv.exceptions.InstallError]:           self.run_command('build')
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/_distutils/cmd.py", line 313, in run_command
[pipenv.exceptions.InstallError]:           self.distribution.run_command(command)
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/dist.py", line 1214, in run_command
[pipenv.exceptions.InstallError]:           super().run_command(command)
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/_distutils/dist.py", line 986, in run_command
[pipenv.exceptions.InstallError]:           cmd_obj.run()
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/_distutils/command/build.py", line 136, in run
[pipenv.exceptions.InstallError]:           self.run_command(cmd_name)
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/_distutils/cmd.py", line 313, in run_command
[pipenv.exceptions.InstallError]:           self.distribution.run_command(command)
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/dist.py", line 1214, in run_command
[pipenv.exceptions.InstallError]:           super().run_command(command)
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/_distutils/dist.py", line 986, in run_command
[pipenv.exceptions.InstallError]:           cmd_obj.run()
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/command/build_ext.py", line 79, in run
[pipenv.exceptions.InstallError]:           _build_ext.run(self)
[pipenv.exceptions.InstallError]:         File "/Users/johnmcvey/.local/share/virtualenvs/hardware-PicR8xfW/lib/python3.7/site-packages/setuptools/_distutils/command/build_ext.py", line 339, in run
[pipenv.exceptions.InstallError]:           self.build_extensions()
[pipenv.exceptions.InstallError]:         File "/private/var/folders/r6/l11jt3f5461f4t1cc_2pmt2c0000gp/T/pip-install-rjbimtoh/pillow_288a2b4a62b14ad08257bd657605fbee/setup.py", line 804, in build_extensions
[pipenv.exceptions.InstallError]:           raise RequiredDependencyException(f)
[pipenv.exceptions.InstallError]:       __main__.RequiredDependencyException: jpeg
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:       During handling of the above exception, another exception occurred:
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:       Traceback (most recent call last):
[pipenv.exceptions.InstallError]:         File "<string>", line 36, in <module>
[pipenv.exceptions.InstallError]:         File "<pip-setuptools-caller>", line 34, in <module>
[pipenv.exceptions.InstallError]:         File "/private/var/folders/r6/l11jt3f5461f4t1cc_2pmt2c0000gp/T/pip-install-rjbimtoh/pillow_288a2b4a62b14ad08257bd657605fbee/setup.py", line 1009, in <module>
[pipenv.exceptions.InstallError]:           raise RequiredDependencyException(msg)
[pipenv.exceptions.InstallError]:       __main__.RequiredDependencyException:
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:       The headers or library files could not be found for jpeg,
[pipenv.exceptions.InstallError]:       a required dependency when compiling Pillow from source.
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:       Please see the install instructions at:
[pipenv.exceptions.InstallError]:          https://pillow.readthedocs.io/en/latest/installation.html
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:       [end of output]
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]:   note: This error originates from a subprocess, and is likely not a problem with pip.
[pipenv.exceptions.InstallError]: error: legacy-install-failure
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]: × Encountered error while trying to install package.
[pipenv.exceptions.InstallError]: ╰─> pillow
[pipenv.exceptions.InstallError]:
[pipenv.exceptions.InstallError]: note: This is an issue with the package mentioned above, not pip.
[pipenv.exceptions.InstallError]: hint: See above for output from the failure.
ERROR: Couldn't install package: pillow
 Package installation failed...
  ☤  ▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉ 0/1 — 00:00:05
make[2]: *** [Makefile:55: setup] Error 1
make[2]: Leaving directory '/Users/johnmcvey/github/opentrons/opentrons/hardware'
make[1]: *** [Makefile:65: hardware-py-setup] Error 2
make[1]: Leaving directory '/Users/johnmcvey/github/opentrons/opentrons'
make: *** [Makefile:61: setup-py] Error 2
```

--- a/DEV_SETUP.md
+++ b/DEV_SETUP.md
@@ -180,7 +180,7 @@ eval "$(pyenv init -)"
 
 #### 3. Install `jpeg` if on ARM Mac (M1)
 
-With the addition of pillow in `/hardware` [jpeg](https://formulae.brew.sh/formula/jpeg) is needed.
+`/hardware` depends on the Python library Pillow. On ARM Macs, `pip` will build Pillow from source, which requires [jpeg](https://formulae.brew.sh/formula/jpeg) to be installed.
 
 ```shell
 brew install jpeg

--- a/DEV_SETUP.md
+++ b/DEV_SETUP.md
@@ -178,6 +178,14 @@ eval "$(pyenv init -)"
 # ...
 ```
 
+#### 3. Install `jpeg` if on ARM Mac (M1)
+
+With the addition of pillow in `/hardware` [jpeg](https://formulae.brew.sh/formula/jpeg) is needed.
+
+```shell
+brew install jpeg
+```
+
 ### Windows
 
 **This section is a work in progress**

